.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
   - sudo apt-get install -y build-essential libcap-dev
 install:
   - pip install tox
+  - pip install -r test_requirements.txt
```

Comment:
Of the two options, I would prefer to rename ``test_requirements.txt`` to ``requirements_test.txt`` in Sideboard, because I like that naming convention better.  That's out of scope for this particular PR, but I'll try to take care of that this week or at least file a ticket for it.  At the same time I'll probably remove the ``test_requires`` line from ``setup.py`` since we never use ``python setup.py test`` anyway.